lab/requirements.md

```diff
@@ -0,0 +1,53 @@
+# OPI Lab Requirements
+
+## Access
+
+- Lab staff can access to all machines.
+- OPI members should access only to their own machines.
+- Remote access to a machine requires private VPN specific to the machine.
+- OPI members cannot access to a machine from another one.
+- SSH connection is available through private VPN.
+- Emergency console can be enabled for remote access in case of issue.
+
+## Backup
+
+- All servers must be backed up.
+- Restoration of servers must be done within a couple of hours.
+- Hosting provider must be trained to do backup/restore.
```

Comment:
No.

Reply:
I agree that not all servers must be backed up. I do say, though, that all critical servers must be backed up. I would like to hear people's view on whether the set of critical servers is the empty set.

Reply:
I would think that some of the critical servers should be backed up, These would be sharable infrastructure services, so that they didn't have to be recreated for every test. e.g. DHCP, TFTP, Web Server (image hosting), Grafana, OTWL GW, sZTP server, etc.

Reply:
What are people's view on the OPI lab providing core infrastructure services, such as the ones @pdp2shirts mentioned, beyond  simply as a place to host ephemeral servers?  As a lab user, I personally would rather have the lab infrastructure provide me with dhcp, ldap, and NFS rather than rolling my own.  And if folks want these lab services, then we need to talk about SLA.